code_generator/Templates/UnityComponentUpdateGenerator.tt

```diff
@@ -0,0 +1,20 @@
+<#@ template language="C#" #>
+<#@ output extension=".cs" #>
+<#
+    var fieldDetailsList = GetFieldDetailsList();
+    var componentDetails = GetComponentDetails();
+    var generatedHeader = CommonGeneratorUtils.GetGeneratedHeader();
+#>
+<#= generatedHeader #>
+
+using Improbable.Gdk.Core;
+
+namespace <#= qualifiedNamespace #>
+{ 
+    public struct <#= componentDetails.TypeName #>Update : ISpatialComponentUpdate
```

Comment:
Please make this a sub class of the existing `<#= componentDetails.TypeName #>` namespacing class

Reply:
So this makes it `SpatialOSPosition.Update`, instead of `SpatialOSPositionUpdate`. Which will be consistent with the other components, and avoid name conflicts

Reply:
Great idea. Will do.